### DISCUSSION TITLE
Stair placement rules work but breaking them deletes every stair in chunk

### DISCUSSION THREAD

**Discussion by JackM33** [Minestom/Minestom#1000] [Help]
Hi so I stumbled upon the stair placement rules and added some to PlacementRules.java and everything works fine in terms of placement, however upon breaking one of them, all of the stairs in that chunk get deleted and a bunch of lighting issues emerge (see screenshot). 

Are the stair placement rules unstable (inferred because they aren't already added to PlacementRules) or could I be doing something the wrong way? Thanks!

![2022-04-28_00 18 23](https://user-images.githubusercontent.com/47346677/165698396-641790b0-0d8e-461f-9d4c-bd64c2a96fb2.png)

After breaking:
![2022-04-28_00 18 49](https://user-images.githubusercontent.com/47346677/165698472-69a43fa7-3bf3-4ab8-a48a-3c1ace0ec439.png)

**Comment by Flamgop** [comment]
The stair placement rules are completely broken, the stairs you're seeing in that chunk are client side (as well as the light), so when you update it (eg. by breaking a block) the client asks to resend the chunk, and realizes the blocks are not there.

**Reply by Flamgop** [reply]
Also if you want to fix the "lighting issues" you can set the `DimensionType`'s light to 2.0f or implement a lighting engine.
